crates/rome_js_analyze/src/analyzers/nursery/use_getter_return.rs

```diff
@@ -0,0 +1,162 @@
+use crate::ControlFlowGraph;
+use rome_analyze::{context::RuleContext, declare_rule, Rule, RuleDiagnostic};
+use rome_console::markup;
+use rome_control_flow::{ExceptionHandlerKind, InstructionKind};
+use rome_js_syntax::{JsGetterClassMember, JsGetterObjectMember, JsReturnStatement};
+use rome_rowan::{AstNode, NodeOrToken, TextRange};
+use rustc_hash::FxHashSet;
+
+declare_rule! {
+    /// Enforces the presence of non-empty `return` statements in getters.
+    ///
+    /// A _getter_ allows defining a property which is dynamically computed.
```

Comment:
I'm not really sure what the purpose of this paragraph is. We should delete it or rephrase it.

I'd help, but I don't understand it with the context of the rule.

Reply:
In fact, this was a try to improve the phrasing of the [ESLint rule](https://eslint.org/docs/latest/rules/getter-return):

> The get syntax binds an object property to a function that will be called when that property is looked up.

The aim is to explain what a getter is.